src/assets/icons/frenchpress.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Did you get this icon somewhere or draw it by yourself? Because license is important.

Also we need to follow the style of other icons (the line weight is important)

Reply:
I grabbed it off an open source site. Let me see if a license was attached to it.

Reply:
I've added my own drawed frenchpress icon to a master branch, you can remove your file and just pull latest changes

Reply:
https://github.com/2brew/2brew.github.io/blob/master/src/assets/icons/frenchpress.svg